x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/space_aware_privilege_section/_privilege_matrix.scss

```diff
@@ -1,14 +0,0 @@
-/**
- * 1. Allow table to scroll both directions
- */
-
-.secPrivilegeMatrix__modal,
```

Comment:
__note:__ looks like only `.secPrivilegeTable__row--isGlobalSpace` is actually used.